libs/ui/elements/src/lib/pagination/pagination.component.html

```diff
@@ -2,17 +2,17 @@
   <button
     (click)="nextPage()"
     [disabled]="currentPage === nPages"
-    class="uppercase text-white flex justify-center items-center p-3 rounded-lg mr-8 cursor-pointer h-14 mr-12 bg-[#8cc630] disabled:bg-gray-100 disabled:cursor-default"
+    class="uppercase text-white justify-center items-center p-3 rounded-lg cursor-pointer h-14 w-44 bg-secondary hidden sm:flex sm:mr-28 disabled:bg-gray-100 disabled:cursor-default"
```

Comment:
Just a remark on the `sm:` classes: I think this is a bit too "context-dependant", as the screen width is not really a strong indication that there is room for the button. To keep in mind if this component has to be reused somewhere!

Reply:
Ok, I see your point. What do you recommend for the case that the screen becomes too narrow to display the "next page" button on the left to the pagination buttons ? Letting the pagination buttons wrap below doesn't really look neat IMO.

Reply:
An endless problem... soon [CSS will allow handling sizes at the block level](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) and not just the page, but for now there is no straightforward way to do this.

A solution might be to leave the button there but make sure that it correctly repositions if there is not enough room, using flex.